packages/react-instantsearch/src/connectors/connectAutoComplete.js

```diff
@@ -0,0 +1,61 @@
+import createConnector from '../core/createConnector';
+import {omit} from 'lodash';
```

Comment:
We do not have an automatic eslint rule for this so for now the consensus in react-instantsearch is to use the {} form. Ultimately it's preferable IF we can then use babel-lodash to automatically transpile to this form, but last time we checked it was not working I think.

No matter what we can discuss this in another issue no prob.